flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/snapshot/RocksDBSnapshotStrategyBase.java

```diff
@@ -92,4 +92,6 @@ public RocksDBSnapshotStrategyBase(
     public String getDescription() {
         return description;
     }
+
+    public void close() {}
```

Comment:
NIT: I'd slightly prefer this method to be abstract to force new implementors to override it.